src/app/pages/ShipyardPage.jsx

```diff
@@ -321,7 +336,7 @@ export default class ShipyardPage extends Page {
             maxWidth: '100%'
           }}
         >
-          <table style={{ width: '12em', position: 'absolute', zIndex: 1 }}>
+          <table style={{ width: '12em', position: 'absolute', zIndex: 1 }} className="shipyard-table">
```

Comment:
Same here. Are double quotes necessary? For strings we usually use single quotes in coriolis.

Reply:
HTML attributes are consistantly double quotes